### PR TITLE
Add key bindings for useful Helm commands

### PR DESCRIPTION
**Pull request by tuhdo**
- helm-info-at-point: find an info entry for symbol at point, if
  available. It can also be used to search for available info pages.

- helm-man-woman: find man pages with Helm.

- helm-locate-library: find a loaded Emacs Lisp library using Helm.

- helm-minibuffer-history: search minibuffer history using Helm.

### CODE REVIEW

File: spacemacs/packages.el [hunk 1]
```diff
@@ -1393,6 +1393,9 @@ ignored)."
         "sl"  'helm-semantic-or-imenu
         "hb"  'helm-bookmarks
         "hl"  'helm-resume
+        "hi"  'helm-info-at-point
+        "hm"  'helm-man-woman
+        "hL"  'helm-locate-library
```
Comment (syl20bnr) [inline spacemacs/packages.el]: If it is related to elisp then it should go behind `SPC m h L` with the other key bindings in `spacemacs/keybindings.el`

**Comment by tuhdo** [comment]
@syl20bnr I fixed it.

**Comment by tuhdo** [comment]
@syl20bnr Is there any more problem with these key bindings?

**Comment by syl20bnr** [comment]
Seems good, I can see that I named badly the macro `spacemacs||describe-set-key`.
I'll merge this tonight.

**Comment by syl20bnr** [comment]
Thank you ! :+1: 
Cherry-picked into `develop` branch, you can safely delete your branch.

**Pull request closed without merging**